tests/mpi/project_bv_div_conf.cc

```diff
@@ -203,7 +203,12 @@ namespace ResFlow
       DoFTools::make_hanging_node_constraints(dof_handler, constraints);
 
       VectorTools::project_boundary_values_div_conforming(
-        dof_handler, 0, FluxBoundaryValues<dim>(), 0, constraints);
+        dof_handler,
+        0,
+        FluxBoundaryValues<dim>(),
+        0,
+        constraints,
+        StaticMappingQ1<dim>::mapping);
```

Comment:
... the same is true for `VectorTools::project_boundary_values_div_conforming` :(